app/office/models.py

```diff
@@ -0,0 +1,64 @@
+from django.apps import apps
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+from django.utils import timezone
+from django_cleanup import cleanup
+from pathlib import Path
+from storages.backends.s3boto3 import S3Boto3Storage
+
+PATH = apps.get_app_config("office").path
+
+
+class ClosureTemplate(models.Model):
+    def _get_call_default():
+        return Path(PATH + "/closure/templates/call_template.txt").read_text()
+
+    def _get_email_default():
+        return Path(PATH + "/closure/templates/email_template.html").read_text()
+
+    def _get_notice_default():
+        return Path(PATH + "/closure/templates/notice_template.html").read_text()
+
+    created_at = models.DateTimeField(default=timezone.now)
+    call_text = models.TextField(default=_get_call_default, blank=False)
+    email_html = models.TextField(default=_get_email_default, blank=False)
+    notice_html = models.TextField(default=_get_notice_default, blank=False)
+
+
+@cleanup.select
+class Closure(models.Model):
+    def _get_template_default():
+        if not ClosureTemplate.objects.exists():
+            ClosureTemplate.objects.create().save()
+        return ClosureTemplate.objects.latest("created_at").pk
```

Comment:
get_or_create works here I think:  [get_or_create](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.get_or_create)

You don't need to call save after create